pkg/eraser/eraser.go

```diff
@@ -22,9 +23,54 @@ const (
 
 var (
 	// Timeout  of connecting to server (default: 10s)
-	timeout time.Duration = 100 * time.Second
+	timeout time.Duration = 10 * time.Second
 )
 
+type Client interface {
+	listImages(context.Context) ([]*pb.Image, error)
+	listContainers(context.Context) ([]*pb.Container, error)
+	removeImage(context.Context) error
+}
+
+type client struct {
+	images  pb.ImageServiceClient
+	runtime pb.RuntimeServiceClient
+}
+
+func (c *client) listContainers(context.Context) (list []*pb.Container, err error) {
```

Comment:
```suggestion
func (client *client) listContainers(context.Context) (list []*pb.Container, err error) {
```

for consistency, or use `c *client` on others